index.html

```diff
@@ -32,6 +32,94 @@ <h1 id="name" class="text">
       <div>
         <img id="shape02" src="images/image_geometry_2.png" >
       </div>
-    </main>
+    </main> 
+    <h2 class="title">
+      Projects
+    </h2>
+    <section id="projects">
+      <div id="pr01">
+        <img id="pr01img" src="/images/pr01img.png">
+        <div class="projectinfo" id="pr_info_01">
+          <div class="projectname">Project name goes here</div>
+          <div class="tags">
+            <div class="tag box1">HTML</div>
+            <div class="tag box2">CSS</div>
+            <div class="tag box3">Javascript</div>
+          </div>
+          <div type="image" id="prdetailbutton"></div>
+        </div>
+        <div id="pr02">
+          <img id="pr02img" src="/images/pr02img.png">
+          <div class="projectinfo" id="pr_info_03">
+            <div class="projectname">Project name goes here</div>
+            <div class="tags">
+              <div class="tag box1">HTML</div>
+              <div class="tag box2">CSS</div>
+              <div class="tag box3">Javascript</div>
+            </div>
+            <div type="image" id="prdetailbutton"></div>
+          </div>
+          <div id="pr03">
+            <img id="pr03img" src="/images/pr03img.png">
+            <div class="projectinfo" id="pr_info_03">
+              <div class="projectname">Project name goes here</div>
+              <div class="tags">
+                <div class="tag box1">HTML</div>
+                <div class="tag box2">CSS</div>
+                <div class="tag box3">Javascript</div>
+              </div>
+              <div type="image" id="prdetailbutton"></div>
+            </div>
+      </div>
+    </section>
+    <h2 class="title">
+      About
+    </h2>
+    <section id="about">
+    <p id="bio">
+      Lorem ipsum dolor sit amet, consectetur<br>
+      adipiscing elit, sed do eiusmod tempor<br>
+      incididunt ut labore et dolore magna<br>
+      aliqua. Ut enim ad minim veniam, quis<br>
+      nostrud exercitation ullamco laboris nisi<br>
+      ut aliquip ea eiusmod
+    </p>
+    <div type="image" id="cvbutton"></div>
+    <div id="techskills">
+      <div class="techbox" id="languages">
+        <ul class="ulheader">Languages</ul>
+        <li class="article">Javascript</li>
+        <li class="article">HTML</li>
+        <li class="article">CSS</li>
+        <img id="langicon" src="images/ic_languages_mob.png">
+      </div>
+      <div class="techbox" id="frameworks">
+        <ul class="ulheader">Frameworks</ul>
+        <li class="article">NaN</li>
+        <li class="article">NaN</li>
+        <li class="article">Nan</li>
+        <img id="frameicon" src="images/ic_framework.png">
+      </div>

```

Comment:
@abdulhamiid hey, I hope you are well. First of all, I appreciate your time. I just want to ask about the difference between "img" and background images that you mentioned in the last article of your suggestions. What should I do exactly? Should I use the "background styled div" tag instead of "img" tag? I just want to make sure I understand the topic. Please let me know when you are available.

Reply:
I already did the other changes. Please give some advice for the last one

Reply:
Hi @s2jack , yes you have to assign to the div a `background-image:url(./image.png)`.